packages/vite/src/node/importGlob.ts

```diff
@@ -182,47 +187,19 @@ function lexGlobPattern(
         throw new Error('unknown import.meta.glob lexer state')
     }
   }
-
-  const endIndex = getEndIndex(code, i)
-  const options = code.substring(i + 1, endIndex)
-  const commaIndex = options.indexOf(`,`)
+  const clearCode = code
+    .slice(i)
+    .replace(singlelineCommentsRE, blankReplacer)
+    .replace(multilineCommentsRE, blankReplacer)
+  const endIndex = clearCode.indexOf(')')
+  const commaIndex = clearCode.indexOf(',')
```

Comment:
I try in the import glob, the code will had longer strings. I think maybe we can do this like `lexGlobPattern` 🤔

Reply:
Hmm, maybe in another PR?